oss.krtirtho.Spotube.yml

```
@@ -0,0 +1,51 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/flatpak/flatpak-builder/master/data/flatpak-manifest.schema.json
+app-id: oss.krtirtho.Spotube
+runtime: org.gnome.Platform
+runtime-version: "40"
```

```suggestion
runtime-version: "41"
```

also do you really need the GNOME runtime? Isn't freedesktop enough?